lexers/t/twig.go

```diff
@@ -10,7 +10,7 @@ var Twig = internal.Register(MustNewLexer(
 	&Config{
 		Name:      "Twig",
 		Aliases:   []string{"twig"},
-		Filenames: []string{},
+		Filenames: []string{"*.twig"},
```

Comment:
Didn't find it in Pygments.

Reply:
This is a wakatime top language: https://wakatime.com/leaders/languages.